wiki/Tournaments/Official_support/en.md

```diff
@@ -2,17 +2,52 @@
 
 The [osu! team](/wiki/People/The_Team) runs a program where they may opt to provide community-run tournaments that abide by a certain set of criteria and expectations with extra support, at their discretion.
 
+For any queries and clarifications, please contact [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
+
 ## Benefits
 
 Officially supported tournaments have access to:
 
-- **Profile badge prizes for first-place winners**
-- Assessment for a main-menu banner during major matches (generally quarterfinals and above), subject to availability and criteria (see the [relevant section](#requesting-main-menu-banner-support) below)
-- A listing in the [automatic tournament registrations system](https://osu.ppy.sh/community/tournaments)
+- [**Profile badge prizes for first-place winners**](#profile-badges)
+- Assessment for a [main-menu banner advertisement](#requesting-main-menu-banner-support) during major matches, subject to availability and criteria
 - A dedicated screening process to prevent players with serious integrity infringements (cheating, multiaccounting, tournament bans) from playing
```

Comment:
```suggestion
- A dedicated screening process to prevent players with serious integrity infringements (cheating, multi-accounting, tournament bans) from playing
```